webapp/src/Controller/API/SubmissionController.php

```diff
@@ -269,6 +270,8 @@ public function addSubmissionAction(Request $request, ?string $id): Response
                     throw new BadRequestHttpException(
                         sprintf("Team %s not found or not enabled", $teamId));
                 }
+
+                $user = $team->getUsers()->first() ?: null;
```

Comment:
Do we want to allow to specify a user_id as well?

Reply:
Do we? Might make sense actually. Do we want to make sure the user is part of the team? I can add it and a test for it

Reply:
We can also do this separately from this PR

Reply:
I do like the idea. I think my proposal makes sense and is easy to implement